assignments/walk_tests_repo.go

```diff
@@ -83,6 +83,12 @@ func readTestsRepositoryContent(dir string, courseID uint64) ([]*qf.Assignment,
 			if err := json.Unmarshal(contents, &benchmarks); err != nil {
 				return nil, "", fmt.Errorf("failed to unmarshal %q: %s", criteriaFile, err)
 			}
+			for _, bm := range benchmarks {
```

Comment:
Could you add a comment to explain why this is necessary... Because it may not be obvious even to us in 3-6 months.

Reply:
Done.